guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2925 @@
+{
```

Comment:
<p>It is good that data collections are sorted into themes while displaying. Thumbs up.</p><p>A question I have here, is that what does <code>2019 Daytime Population (Esri)</code>  mean? also is <code>'2024 Population by Generation (Esri)'</code>  a projected survey? Maybe it would be good to show the link to esri help doc where these field categories are explained...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>I tried to look for the docs that explain these field categories but could not find something that explains the categories. I will keep looking.</p>